blockade/api/manager.py

```diff
@@ -14,59 +14,95 @@
 # limitations under the License.
 #
 
+from blockade.config import BlockadeConfig
 from blockade.core import Blockade
+from blockade.errors import BlockadeNotFound
 from blockade.errors import InvalidBlockadeName
 from blockade.net import BlockadeNetwork
 from blockade.state import BlockadeState
 
-# TODO(pdmars): breaks if server restarts, refactor to be part of BlockadeState
-BLOCKADE_CONFIGS = {}
+import os
+import yaml
+
+
 DATA_DIR = "/tmp"
+BASE_BLOCKADE_DIR = os.path.join(DATA_DIR, ".blockade")
+REST_STATE_FILE = os.path.join(BASE_BLOCKADE_DIR, "rest_state.yaml")
 
 
 class BlockadeManager:
-    """Simple helper for what should eventually be persisted via BlockadeState
+    """Simple helper to persist Blockade configurations managed via REST API
     """
 
     @staticmethod
     def set_data_dir(data_dir):
         global DATA_DIR
         DATA_DIR = data_dir
+        BASE_BLOCKADE_DIR = os.path.join(DATA_DIR, ".blockade")
+        REST_STATE_FILE = os.path.join(BASE_BLOCKADE_DIR, "rest_state.yaml")
```

Comment:
I think you need to declare these two as global if the changes are to stick. However, is there a reason to use global vars at all? Perhaps all three of these vars could be on the class instead?